ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -249,6 +252,21 @@ module.exports = Marionette.LayoutView.extend({
       homeBoundingBox !== undefined ? homeBoundingBox : defaultHomeBoundingBox
     )
   },
+  addPanZoom: function() {
+    const self = this
+    const PanZoomView = Marionette.ItemView.extend({
+      template() {
+        return (
+          <Gazetteer setState={({ polygon }) => self.map.doPanZoom(polygon)} />
```

Comment:
❗️ I think you need to add doPanZoom() to the list of functions that must be implemented [map.js](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.js)

Reply:
This will be a good follow-up step. But since we aren't implementing any new map providers in the next week or two, and the interface isn't enforced by TypeScript, I think it's safe to punt this and the other feedback to the next PR.